nf_core/modules/lint/main_nf.py

```diff
@@ -212,22 +212,18 @@ def check_process_section(self, lines):
 
     for l in lines:
         l = l.strip()
-        l = l.replace('"', "")
-        l = l.replace("'", "")
+        l = l.lstrip('"')
+        l = l.lstrip("'")
```

Comment:
Do we need any of this with the new regexes?

Reply:
If we do, could at least condense it to a single call:

```python
l = l.strip(" '\"")
```

Reply:
I think we need to remove at least the first quote for the comparision with startswith() later, I will join them to a single call